initial_setup/tui/tui.py

```diff
@@ -1,28 +1,32 @@
 from pyanaconda.ui.tui import TextUserInterface
 from pyanaconda import threading
 
-from initial_setup.product import product_title, is_final
+from initial_setup.product import product_title, is_final, eula_available
 from initial_setup.common import list_usable_consoles_for_tui
+from initial_setup.i18n import _, N_
 from .hubs import InitialSetupMainHub
 
 from simpleline import App
 
 import os
 import sys
-import gettext
 import select
 import contextlib
 import termios
 import logging
 log = logging.getLogger("initial-setup")
 
-# localization
-_ = lambda x: gettext.ldgettext("initial-setup", x)
-N_ = lambda x: x
-
-QUIT_MESSAGE = N_("Are you sure you want to quit the configuration process?\n"
+def get_quit_message():
+    if eula_available():
+        return N_("Are you sure you want to quit the configuration process?\n"
+                  "You might end up with an unusable system if you do. Unless the "
+                  "License agreement is accepted, the system will be rebooted.")
+    else:
+        return N_("Are you sure you want to quit the configuration process?\n"
                   "You might end up with unusable system if you do.")
```

Comment:
Is this the same method as in GUI?

Reply:
That's a good point, both strings are identical & could be shared. I'll try to do that.